pjsip/include/pjsua-lib/pjsua.h

```diff
@@ -7194,6 +7194,20 @@ typedef struct pjsua_snd_dev_param
      */
     unsigned		mode;
 
+    /*
+     * When opening the sound device, the settings set from
+     * #pjsua_snd_set_setting() will be applied to any sound device.
```

Comment:
Modify the doc to explain that the setting is not just from pjsua_snd_set_setting(), but also taken from the first sound device opened. As in the original report, the latter is more problematic because app is not aware since it never calls any API to set the sound dev settings.